src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/LoggingSelectionInfo.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections.Generic;
+using Microsoft.UI.Xaml.Data;
+
+public class LoggingSelectionInfo : ISelectionInfo
```

Comment:
To be valid, this class should also implement `ICollectionView` (i.e. log methods like you did)